packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -59,13 +59,12 @@ export class TSClient implements Generatable {
       generator,
       sqliteDatasourceOverrides,
       outputDir,
-      schemaDir,
+      schemaPath,
       runtimeDir,
       runtimeName,
       datasources,
       dataProxy,
     } = this.options
-    const schemaPath = path.join(schemaDir, 'schema.prisma')
```

Comment:
Because the schema path is hardcoded here, and because we read it here in order to inline it, this would lead to schema not being found if it sits in a custom path. Now instead, we use the value from the CLI.